Neos.EventSourcedContentRepository/Classes/Domain/Context/NodeAggregate/Event/NodeAggregateWithNodeWasAutoCreated.php

```diff
@@ -25,181 +24,165 @@
 use Neos\Flow\Annotations as Flow;
 
 /**
- * Node aggregate with node was created event
+ * A node aggregate with its initial node was auto created
```

Comment:
so that's a "auto-created child node" of a node?

Is that already supported by the graph projector? :) (didn't check myself yet)

Reply:
Exactly, and probably not. I'm not even sure whether we need separate events for this but if we do, we'll see a lot more of them (e.g. SubnodeSpecializationVariantWasCreateed)

Reply:
- Idea: Node is *Tethered*
- property of NodeAggregate